config/galaxy.yml.sample

```diff
@@ -1,21 +1,21 @@
 # Galaxy is configured by default to be usable in a single-user development
```

Comment:
this file is [generated](manage-script-name) from [schema](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/config_schema.yml).